tests/test_pfif.py

```diff
@@ -56,7 +56,7 @@ class Entity:
     return entity
 
 
-class TestCase:
+class _TestCase:
```

Comment:
Can you comment the reason of "_" since it's a bit unusual naming?

Can be workarounded in other namings like `PFIFTestCase` or `PersonForTest`.

Reply:
The underscore was just meant as Python's "private" signifier; it made sense to me since the classes are private to the test. I've switched them to the more specific names you suggested though.

Reply:
I see. I haven't seen the underscore used for class names, but that sounds reasonable. You can leave the underscore too if you prefer.